services/121-service/src/migration/test-migration.sh

```diff
@@ -31,7 +31,7 @@ docker stop "$DOCKER_DB_NAME"
 docker rm "$DOCKER_DB_NAME"
 
 # Start the specified Docker container to apply the migration and load some data, then stop it again
-npm run start:services
+npm run start:services:detach
```

Comment:
This change was needed to make this script work again. Would be better as a seperate PR, but meh